index.html

```diff
@@ -186,6 +227,13 @@ <h2 itemprop="description">Save your 2FA secrets, then use this to scan them aga
     var counter = $("#counter").val() || "0";
     uri += `&counter=${counter}`;
   }
+  if ( adv_options ) {
```

Comment:
Can you remove the extra spaces around `adv_options` to keep with the existing formatting?

Reply:
Fixed with: e5851b499a7c6dda17b69ea0d791b2aaf7f6bdf7